### ISSUE TITLE
sliderTextInput compatibility problem on Quarto doc in the argument `choices`.

### ISSUE THREAD

**Issue by pingfan-hu** [dreamRs/shinyWidgets#689]
The `sliderTextInput` function works out fine on ShinyApp, like this:
```{r}
sliderTextInput(
   inputId = "Id095",
   label = "Choose a letter:", 
   choices = c("a", "b", "c", "d", "e")
)
```
However, it has compatibility problem on Quarto doc. The `choices` argument should be used to define the character vector to select a value from, but Quarto doc doesn't recognizes it and keeps showing a default range from 10 to 100, not matter what range I set, or if I use character vector instead. Below is my example:
```{r}
sd_question(
  name        = 'climate_care',
  type        = 'slider',
  required    = TRUE,
  label       = "**To what extent do you believe human-caused climate change is real?**",
  # Here `option` is a parameter of the argument `choices`
  option      = c("Don't Believe"    = "dont_believe",
                  "Somewhat Believe" = "somewhat",
                  "Neutral"          = "neutral",
                  "Believe"          = "believe",
                  "Strongly Believe" = "strongly_believe"),
  grid        = FALSE,
  force_edges = TRUE,
  width       = "60%"
)
```

Here is a screenshot of a successful implementation on the ShinyApp:
![ShinyApp Works Fine](https://github.com/dreamRs/shinyWidgets/assets/84371500/0f74eaef-9a3a-45d1-a2b2-6c6b8decd499)

Here is another screenshot of Quarto Doc. You can see the range is set as from 10 to 100, which is probably a default value. I tried to set a custom range with a character vector but it was ignored:
![Quarto Doc Has Error on the Range](https://github.com/dreamRs/shinyWidgets/assets/84371500/b860a8f7-be1c-43c1-a15a-dc4cae290590)

If anyone encountered with the same issue, or if you have a solution, please comment below. I appreciate your help!

**Comment by etschste** [comment]
I encountered the same problem.

**Comment by pingfan-hu** [comment]
Yeah this is a relatively rare problem and will only occur when you want to use shiny as server on your `qmd` file. In usual cases people use the ShinyApp and it works out fine. Hopefully the shinyWidgets team could provide some guidance  to us.
